testkit/src/main/scala/org/bitcoins/testkit/fixtures/LndFixture.scala

```diff
@@ -19,30 +18,28 @@ trait LndFixture extends BitcoinSFixture {
   }
 
   def withLnd(test: OneArgAsyncTest): FutureOutcome = {
-    val bitcoindInstance = BitcoindRpcTestUtil.v21Instance()
-    val bitcoind = BitcoindV21RpcClient(bitcoindInstance)
-    val client = LndRpcTestClient.fromSbtDownload(Some(bitcoind))
-
     makeDependentFixture[LndRpcClient](
       () => {
         for {
+          bitcoind <- cachedBitcoindWithFundsF
           _ <- bitcoind.start()
+
+          client = LndRpcTestClient.fromSbtDownload(Some(bitcoind))
           lnd <- client.start()
         } yield lnd
       },
       { lnd =>
         for {
           _ <- lnd.stop()
-          _ <- bitcoind.stop()
         } yield ()
       }
     )(test)
   }
 }
 
 /** A trait that is useful if you need Lnd fixtures for your test suite */
-trait DualLndFixture extends BitcoinSFixture {
-  _: BitcoinSAsyncFixtureTest =>
+trait DualLndFixture extends BitcoinSFixture with CachedBitcoindV21 {
+  _: BitcoinSAsyncFixtureTest with CachedBitcoindFunded[_] =>
```

Comment:
So in plain english this is what this means
```scala
trait DualLndFixture extends BitcoinSFixture with CachedBitcoindV21 {
  _: BitcoinSAsyncFixtureTest with CachedBitcoindFunded[_] =>
```

>I've created a trait called `DualLndFixture` that extends `BitcoinSFixture with CachedBitcoindV21`. It _requires_ a `BitcoinSAsyncFixtureTest` AND `CachedBitcoindFunded[_]` to be injected into this trait before it can be used.

This doesn't make much sense, because you _already_ have a `CachedBitcoindV21` extended, so why do we need to inject _another_ `CachedBitcoindFunded[_]`. Its a bit redundant. 

you probably want to do something like this

```scala
trait DualLndFixture extends BitcoinSFixture with BitcoindFixturesFundedCachedV21
``` 

and then i don't think you need any `{_: BitcoinSAsyncFixtureTest with CachedBitcoindFunded[_] =>` 

Although I didn't run this locally to test it myself